site/guide/working-with-model-inventory.qmd

```diff
@@ -0,0 +1,51 @@
+---
+title: "Working with the model inventory"
+date: last-modified
+listing:
+  - id: model-inventory-listing
+    type: grid
+    max-description-length: 250
+    sort: false
+    fields: [title, description]
+    contents:
+      - register-models-in-model-inventory.qmd
+      - edit-model-inventory-fields.qmd
+      
+---
+The model inventory holds all models. Within the model inventory, register new models and edit fields. Organize the inventory by searching, filtering and sorting models. Configure the inventory by managing column layout and choosing card or table view.
+
+## Register models and edit fields
```

Comment:
I would change `Register models and edit fields` to `What's next` and put it at the very BOTTOM of this landing page, just like in the [Working with model documentation](http://localhost:3663/guide/working-with-model-documentation.html#whats-next) landing

Reply:
I agree with adding a `What's next` at the bottom, but I'm a bit stuck on putting `Register models` at the bottom, when it's a prerequisite to the contents of this guide. At the same time, all users are loaded in with the demo models, so maybe it's okay?

Reply:
That's a good point. 

Maybe register can be a prerequisite? ;) Then in the `What's next` section, you can just link to "edit model inventory fields" instead of complicating it with a card.